src/integration-test/resources/db/JurorManagementController_poolsAtCourtLocation.sql

```diff
@@ -129,3 +133,6 @@ INSERT INTO juror_mod.appearance (attendance_date,juror_number,pool_number,loc_c
 
 INSERT INTO juror_mod.appearance (attendance_date,juror_number,pool_number,loc_code,f_audit,time_in,time_out,travel_time,appearance_stage,non_attendance) VALUES
 	(current_date,'641500005','415230101','415',123456789,'09:30:00',null,'01:12','CHECKED_IN',false);
+
+INSERT INTO juror_mod.appearance (attendance_date,juror_number,pool_number,loc_code,f_audit,time_in,time_out,travel_time,appearance_stage,non_attendance) VALUES
+	(current_date,'641500007','415230101','415',123456789,'09:30:00',null,'01:12',null,false);
```

Comment:
minor point, but if this is a not_in_attendance record then should time_in and travel_time also be null to better reflect live data?